parsers/BR.py

```diff
@@ -96,9 +96,39 @@ def fetch_production(country_code = 'BR', session = None):
 
     return datapoint
 
+def fetch_exchange(country_code1='BR', country_code2='UY', session=None):
+    """Requests the last known power exchange (in MW) between two regions
+    Arguments:
+    country_code1           -- the first country code
+    country_code2           -- the second country code; order of the two codes in params doesn't matter
+    session (optional)      -- request session passed in order to re-use an existing session
+    Return:
+    A dictionary in the form:
+    {
+      'sortedCountryCodes': 'DK->NO',
+      'datetime': '2017-01-01T00:00:00Z',
+      'netFlow': 0.0,
+      'source': 'mysource.com'
+    }
+    where net flow is from DK into NO
+    """
+
+    gd = get_data()
+
+    data = {
+        'datetime': gd['Data'],
+        'sortedCountryCodes': 'BR->UY',
```

Comment:
Can you convert this to the form I've used for 'datetime' in fetch_production?
arrow.get(gd['Data']).datetime should be fine.

Reply:
Yes indeed. You need `gd['Data'].datetime`.
Also, are we sure that the netFlow you're getting is indeed BR->UY? i.e. export from Brazil?

Reply:
Yeah, I was confused about that, because it's a negative value that means the energy that comes to Brazil. I changed that.

About the data, when I change to `gd['Data'].datetime` it throws:

`'unicode' object has no attribute 'datetime'`

I changed to parse the data. Check if is good in the new commit